mon-pix/app/components/challenge-item-qroc.js

```diff
@@ -32,7 +32,7 @@ export default class ChallengeItemQroc extends ChallengeItemGeneric {
 
   _getAnswerValue() {
     const qcrocProposalAnswerValueBis =
```

Comment:
```suggestion
    const qrocProposalAnswerValueBis =
```

à changer aussi dans le `return` juste après